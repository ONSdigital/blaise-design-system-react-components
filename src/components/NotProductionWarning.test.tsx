import React from "react";
import { render, screen } from "@testing-library/react";
import NotProductionWarning from "./NotProductionWarning";

describe("ONS Not production warning Test", () => {
    it("matches Snapshot", () => {
        expect(<NotProductionWarning />).toMatchSnapshot();
    });

    it("should display warning paragraph text", () => {
        render(<NotProductionWarning />);
        expect(screen.getByText(/This environment is not a production environment. Do not upload any live data to this service./i)).toBeVisible();
    });
});