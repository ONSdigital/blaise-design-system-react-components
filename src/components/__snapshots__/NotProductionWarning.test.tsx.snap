// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ONS Not production warning Test matches Snapshot 1`] = `<NotProductionWarning />`;
